From 790b02c9b328d36b8cef34f27a8b02fe911e6ff5 Mon Sep 17 00:00:00 2001
From: Even Xu <Feng.Xu@freescale.com>
Date: Mon, 13 Feb 2012 15:10:15 +0800
Subject: [PATCH 1023/2463] ENGR00174316 MX6Q ARM2: Fix ov5640_mipi IOMUX
 incorrect configure

One type error on ov5640_mipi IOMUX configure, fix it.

Signed-off-by: Even Xu <b21019@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_arm2.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_arm2.c b/arch/arm/mach-mx6/board-mx6q_arm2.c
index 5ea5a2a..77865dd 100644
--- a/arch/arm/mach-mx6/board-mx6q_arm2.c
+++ b/arch/arm/mach-mx6/board-mx6q_arm2.c
@@ -575,7 +575,7 @@ static void mx6_mipi_sensor_io_init(void)
 
 	if (cpu_is_mx6q()) {
 		mipi_sensor_pads = mx6q_arm2_mipi_sensor_pads;
-		mipi_sensor_pads_cnt = ARRAY_SIZE(mx6dl_gpmi_nand);
+		mipi_sensor_pads_cnt = ARRAY_SIZE(mx6q_arm2_mipi_sensor_pads);
 	} else if (cpu_is_mx6dl()) {
 		mipi_sensor_pads = mx6dl_arm2_mipi_sensor_pads;
 		mipi_sensor_pads_cnt = ARRAY_SIZE(mx6dl_arm2_mipi_sensor_pads);
-- 
1.7.7.4

